### PR TITLE
add script for regression test case OCP-41642

### PR DESCRIPTION
**Pull request by qiliRedHat**
OCP-41642 - Load cluster to test API priority and fairness
https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-41642

The scripts for this test case was on Eric's personal fork, now move it to svt repo.
cd svt/openshift_scalability/ && wget https://github.com/RH-ematysek/svt/blob/4968d67f253cfdd4b948df2b6faf3d965f0f033e/openshift_scalability/run_apf.sh 
cd config && wget https://github.com/RH-ematysek/svt/blob/4968d67f253cfdd4b948df2b6faf3d965f0f033e/openshift_scalability/config/apf_template.yaml && cd ..

### CODE REVIEW

**Comment by qiliRedHat** [comment]
@paigerube14 PTAL

**Comment by paigerube14** [comment]
Thought, should we go ahead and just add this as a custom workload under the perfscale regression ci folder that you are working on in https://github.com/openshift/svt/pull/720/files?

File: openshift_scalability/run_apf.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+for ((i=0; i<NUM_PROJECTS; i++)); do
+  oc delete project "${BASENAME}${i}"
+done
+
+date
+
+
+# Build config Init:Error, I think the script needs update to get from a git that is reachable
+# build.build.openshift.io/buildconfig0-1   Source   Dockerfile,Git   Failed (FetchSourceFailed)   3 minutes ago   3m31s
```
Comment (paigerube14) [inline openshift_scalability/run_apf.sh:33]: think we can use something like this image instead ot get around? 
quay.io/openshifttest/pause:latest

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Comment by qiliRedHat** [comment]
> Thought, should we go ahead and just add this as a custom workload under the perfscale regression ci folder that you are working on in https://github.com/openshift/svt/pull/720/files?

Paige, I prefer to save his origin code here as a reference and enhance it later.

**Pull request merged**